geometry_msgs/PoseStamped goal
---
int64 err_no

string err_msg
